docs/tutorials/configure/pet-photographer.md

```diff
@@ -0,0 +1,984 @@
+
+![The vision service configuration panel showing the color set to blue, the hue tolerance set to 0.06, and the segment size set to 100.](/tutorials/pet-photographer/vision-service.png)
+
+For more detailed information, refer to [Configure a color detector](/services/vision/detection/#configure-a-color_detector).
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
+
+Add the vision service object to the services array in your rover’s raw JSON configuration:
+
+```json {class="line-numbers linkable-line-numbers"}
+  {
+    "name": "my_color_detector",
+    "type": "vision",
+    "model": "color_detector",
+    "attributes": {
+      "segment_size_px": 100,
+      "detect_color": "#43a1d0",
+      "hue_tolerance_pct": 0.06
+    }
+  },
+  ... // Other services
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Configure the color filter camera
+
+With the vision and data management services configured, you can now configure your camera to filter by color and store photos to Viam's cloud.
+
+### Configure your camera
+
+Navigate to your robot's page in the app and click on the **Config** tab.
+
+1. Add your robot's [camera](/components/camera/) as a component by clicking **Create component** in the lower-left corner of the page and entering 'webcam'.
+1. Select the `webcam` model and enter 'cam' as the name for your camera.
+1. Then click **Create**.
+
+Your robot's configuration page now includes a panel for your camera.
+
+- To choose the camera the robot will use, click the **video path** field.
+  - If your robot is connected, you'll see a list of available cameras.
+  - Select the camera you want to use, then click **Save config**.
+
+![An instance of the webcam component named 'cam'](/tutorials/pet-photographer/webcam-component.png)
+
+### Add colorfilter component
+
+1. Click the **Components** subtab and then click **Create component**.
+1. Next, select the `local modular resource` type from the list.
+   {{<imgproc src="/tutorials/pet-photographer/add-local-module-select.png" resize="300x" declaredimensions=true alt="The add a component modal showing the list of components to add with 'local modular resource' shown at the bottom">}}
+1. On the following screen:
+
+   1. Select the camera from the drop down menu.
+   1. Enter the {{< glossary_tooltip term_id="model-namespace-triplet" text="model namespace triplet">}} for your modular resource's {{< glossary_tooltip term_id="model" text="model" >}}, `example:camera:colorfilter`.
+   1. Provide a name for this instance of your modular resource.
+      This name must be different from the module name.
+
+      {{<imgproc src="/tutorials/pet-photographer/add-colorfilter-module-create.png" resize="400x" declaredimensions=true alt="The add a component model showing the create a module step for a local color filter module">}}
+
+1. Click **Create** to create the modular resource component.
+1. Copy the following JSON configuration into the **Attributes** section:
```

Comment:
Nit: I like to use the following language for "in the brand new pane that just poofed in front of you:

```suggestion
1. In the resulting module configuration pane, copy the following JSON configuration into the **Attributes** section:
```

Opt, up to you.